filebeat/config/config.go

```diff
@@ -40,14 +40,14 @@ var (
 )
 
 const (
-	LogInputType   = "log"
-	StdinInputType = "stdin"
+	LogType   = "log"
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported const LogType should have comment (or a comment on this block) or be unexported